ZipPaymentWidget.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { requireNativeComponent, StyleSheet } from 'react-native'
+
+class ZipPaymentWidget extends React.Component {
```

Comment:
Can we use function component instead?  So that

1. We keep things consistent.
2. Follow the direction React is pushing the community towards.

https://beta.reactjs.org/reference/react/Component